.thoth.yaml

```diff
@@ -7,7 +7,7 @@ runtime_environments:
     operating_system: 
       name: rhel
```

Comment:
Should I change to `ubi`?

Reply:
No need to do that - rhel/ubi are basically synonyms here.